examples/traffic-light.manifold

```diff
@@ -0,0 +1,36 @@
+* = import Manifold.core
+* = import Manifold.digital
+
+Type TrafficLight = (
+	green:Boolean  = false,
+	yellow:Boolean = false,
+	red:Boolean    = false
+)
+
+GREEN_SECONDS  = 10
+YELLOW_SECONDS = 3
+TOTAL_SECONDS  = 2 * (GREEN_SECONDS + YELLOW_SECONDS)
+
+seconds = Manifold.Digital.count(hz: 1, mod: TOTAL_SECONDS)
+
+if (seconds < GREEN_SECONDS) {
+  north_light = south_light = (green: true)
+  east_light = west_light = (red: true)
+
+} else if (seconds < GREEN_SECONDS + YELLOW_SECONDS) {
```

Comment:
The problem with making hardware from conditionals is that this breaks as soon as we go to a different domain. This leads to issues in compilation because what represents a conditional in each domain changes (e.g. it might be a switch in microfluidics, or an inhibitory connection in neural nets, etc.).